base/setup/usetup/lang/pt-PT.h

```diff
@@ -1810,7 +1810,7 @@ static MUI_ENTRY ptPTDeletePartitionEntries[] =
     {
         8,
         21,
-        "\x07  Para retornar \205 tela anterior sem excluir",
+        "\x07  Para retornar \205 ecrã anterior sem excluir",
```

Comment:
Unicode characters must be converted with code7bit utility in the scope of this module.